### ISSUE TITLE
Connection fails when using longer static challenge/MFA response

### ISSUE THREAD

**Issue by Connatus** [OpenVPN/openvpn-gui#317]
My organization uses a MFA system that uses 64 character long responses for the token
This generates an error when attempting to use the GUI, but works fine when using the command line version of openvpn or other openvpn clients.
I receive this error in the window.
![image](https://user-images.githubusercontent.com/4542086/68699239-a6ab7b00-0550-11ea-8052-5c67196941dd.png)

and this error appears in the log
```
Tue Nov 12 13:12:15 2019 MANAGEMENT: CMD 'username "Auth" "username"'
Tue Nov 12 13:12:15 2019 MANAGEMENT: CMD 'password [...]'
Tue Nov 12 13:12:15 2019 MANAGEMENT: CMD 'aHVyYmduYmlydXV2bGtkbg=="'
Tue Nov 12 13:12:45 2019 MANAGEMENT: Client disconnected
Tue Nov 12 13:12:45 2019 ERROR: could not read Auth username/password/ok/string from management interface
Tue Nov 12 13:12:45 2019 Exiting due to fatal error

```

**Comment by selvanair** [comment]
64 character response should work. OpenVPN line parser can handle up to 256 chars in general and the GUI can read up to 128 chars in username and password.

Which version of the GUI? Looks like either the username or password contains some characters the GUI is choking on. Embedded new line characters could cause this error on some old versions. 

Looking at the log snippet, OpenVPN line parser is complaining  that it could not find a closing quote and the log  entry with MANAGEMENT: CMD  'aHVyYmduYmlydXV2bGtkbg=="'  shows responses are getting broken up. In case you are doing cut and paste on to the GUI dialog, please double check the entry. Windows has a tendency to  mangle up cut-and-paste.

By the way, if that long reponse is not a secret, sending it using the challenge response (static or dynamic challenge) dialog may be better user experience.

**Comment by Connatus** [comment]
I'm running the latest available release, as far as I know:
![image](https://user-images.githubusercontent.com/4542086/68712037-debeb800-0568-11ea-9902-becae2a1254c.png)
Updated to the latest before filing the issue.

Not currently cutting and pasting anything into the fields. The MFA token acts as a keyboard and enters directly into the field. The MFA token is all alphabetic charaters until the end which is a "TAB" which just switches the fields. There shouldn't be any newline charters either. 

I am currently using static-challenge dialog as well

**Comment by selvanair** [comment]
If you are using static challenge dialog, it can't be illegal characters as the passwd and response are packaged into a base64 blob by the GUI before passing on. There is currently a limitation of about 150 bytes of password + response max due to base64 overhead and the way management parses the password, but 64 bytes should easily work.

Possibly there is a bug -- let me try to reproduce this.

**Comment by selvanair** [comment]
Could you please test the binary included in the test release linked to below?

https://github.com/selvanair/openvpn-gui/releases/tag/issue-317

Thanks.

**Comment by Connatus** [comment]
Great that works! Thanks so much.

**Comment by selvanair** [comment]
Thanks for test and glad it works. Let's keep this open until the fix is merged.

**Comment by selvanair** [comment]
Fixed by 307bb34fa98a97905d9609b3337c1cd8cca1258a
